disk/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # CHANGELOG - disk
 
+## 4.10.0 / 2023-07-07
+
+***Added***:
+
+* Add percentage-based utilization metrics. See [#15138](https://github.com/DataDog/integrations-core/pull/15138).
+
+***Fixed***:
+
+* Bump python version from py3.8 to py3.9. See [#14701](https://github.com/DataDog/integrations-core/pull/14701).
+* [AGENT-9736] Added debug logs. See [#14650](https://github.com/DataDog/integrations-core/pull/14650).
```

Comment:
Is this something we should start doing now?

Reply:
No we should remove all instances of this in the entries